{"exception_type": "<class 'd3m.exceptions.InvalidArgumentTypeError'>", "template": "regression_with_feature_selection", "step": -1, "python_path": "", "exception_message": "Traceback (most recent call last):\n  File \"/user_opt/dsbox/dsbox-ta2/python/dsbox/combinatorial_search/ConfigurationSpaceBaseSearch.py\", line 179, in evaluate_pipeline\n    evaluation_result = self._evaluate(configuration, cache, dump2disk)\n  File \"/user_opt/dsbox/dsbox-ta2/python/dsbox/combinatorial_search/ConfigurationSpaceBaseSearch.py\", line 208, in _evaluate\n    pipeline = self.template.to_pipeline(configuration)\n  File \"/user_opt/dsbox/dsbox-ta2/python/dsbox/template/template.py\", line 137, in to_pipeline\n    return self._to_pipeline(binding, sequence)\n  File \"/user_opt/dsbox/dsbox-ta2/python/dsbox/template/template.py\", line 370, in _to_pipeline\n    data=hyper[hyperName])\n  File \"/src/d3m/d3m/metadata/pipeline.py\", line 623, in add_hyperparameter\n    hyperparams.configuration[name].validate(data)\n  File \"/src/d3m/d3m/metadata/hyperparams.py\", line 2247, in validate\n    super().validate(value)\n  File \"/src/d3m/d3m/metadata/hyperparams.py\", line 294, in validate\n    value=value, for_name=self._for_name(), structural_type=self.structural_type,\nd3m.exceptions.InvalidArgumentTypeError: Value 'huber' for hyper-parameter 'loss' is not an instance of the structural type: typing.Dict\n"}